controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1509,8 +1509,8 @@ bool master_thread::handle_cmdu_1905_higher_layer_data_message(const std::string
 
     const auto protocol       = tlvHigherLayerData->protocol();
     const auto payload_length = tlvHigherLayerData->payload_length();
-    LOG(DEBUG) << "protocol: " << std::hex << protocol;
-    LOG(DEBUG) << "payload length: " << int(payload_length);
+    LOG(DEBUG) << "Protocol: " << std::hex << protocol;
```

Comment:
These changes should be done in another commit.

Reply:
fixed.